pyfar/_concepts/arithmetic_operations.py

```diff
@@ -10,22 +10,34 @@
 :py:func:`~pyfar.classes.audio.FrequencyData` instances can be added in the
 time domain by
 
->>> result = pyfar.classes.audio.add((signal_1, signal_2), 'time')
+>>> result = pf.add((signal_1, signal_2), 'time')
 
 and in the frequency domain by
 
->>> result = pyfar.classes.audio.add((signal_1, signal_2), 'freq')
+>>> result = pf.add((signal_1, signal_2), 'freq')
 
 **Note** that frequency domain operations are performed on the raw spectrum
 ``signal.freq_raw``.
 
-Arithmetic operations also work with more than two instances and supports array
-likes and scalar values, e.g.,
+Arithmetic operations also work with more than two instances and support
+array likes and scalar values, e.g.,
 
->>> result = pyfar.classes.audio.add((signal_1, 1), 'time')
+>>> result = pf.add((signal_1, 1), 'time')
 
 In this case the scalar `1` is broadcasted, i.e., it is is added to every
 sample of `signal` (or every bin in case of a frequency domain operation).
+The shape of arrays need to match the ``cshape`` of the resulting audio object,
+e.g.,
+
+>>> x = np.arange(2 * 3 * 4).reshape((2, 3, 4))
+>>> y = pf.Signal(np.ones((2, 3, 4, 10)), 44100)
+>>> z = pf.add((x, y))
+
+or are broadcasted, e.g.,
+
+>>> x = np.arange(3 * 4).reshape((3, 4))
+>>> y = pf.Signal(np.ones((2, 3, 4, 10)), 44100)
```

Comment:
To clarify, maybe add a note that y is a signal with `signal.cshape = (2, 3, 4)` and a length of 10 samples.